vsproject/build_rustc_minicargo_139.cmd

```diff
@@ -0,0 +1,41 @@
+@setlocal

```

Comment:
Might be an idea to just `@echo off` to avoid prefixing every line (and turn it back on when it's time to run the actual build commands)